lib/vc.js

```diff
@@ -229,7 +240,7 @@ async function createPresentation(options = {}) {
   }
 
   const presentation = options.presentation || {
-    '@context': options.verifiableCredential['@context'],
+    '@context': _extractContext(options.verifiableCredential),
```

Comment:
I don't think we should be reordering contexts for people, rather, we should throw an error if the ordering isn't valid.

Reply:
I can explain! :)

Reply:
Fixed, no longer doing that